package.json

```diff
@@ -7,7 +7,7 @@
     "OneCX Development Team <onecx-development-team@onecx.org>"
   ],
   "config": {
-    "openapiYaml": "src/assets/api/openapi-bff.yaml",
+    "openapiYaml": "https://raw.githubusercontent.com/onecx/onecx-announcement-bff/main/src/main/openapi/openapi-bff.yaml",
```

Comment:
why?

Reply:
@HenryT-CG We discussed with @andrejpetras that it would be better to treat the BFFs OpenAPI spec as the single source of truth, instead of always manually copying the latest version of the file. Specifying the remote URL instead of the relative path will instruct the CLI to pull the latest version of the OpenAPI before each generation.

Reply:
As far as I understood, this is supposed to become the new standard approach in all of our repos in the future.

Reply:
As far as I know: There is no common communication about this "new standard approach". A standard is not set when someone use new things without alignment and communication!

Reply:
I'll revert this change to avoid any confusion. @SchettlerKoehler @andrejpetras @HenryT-CG Lets have a discussion to decide if we want to reapply this change in the future.